frontend/model/contracts/chatroom.js

```diff
@@ -246,7 +245,7 @@ sbp('chelonia/defineContract', {
         }
         Vue.delete(state.users, member)
 
-        if (!state.saveMessage) {
+        if (!state.saveMessage || state.attributes.type === CHATROOM_TYPES.INDIVIDUAL) {
```

Comment:
So yeah, in this PR there's a bunch of references to `CHATROOM_TYPES.INDIVIDUAL`, but per the other comment, it seems like `CHATROOM_TYPES.INDIVIDUAL` and `CHATROOM_TYPES.GROUP` will be made the same, thus removing the need for this. Perhaps this could be done in this PR, instead of doing it in another PR?

Reply:
I think you misunderstood the usage and meaning of `CHATROOM_TYPES.INDIVIDUAL` and `CHATROOM_TYPES.GROUP`. It's not something to decide if the chatroom could let 2 or more people join to.
https://github.com/okTurtles/group-income/pull/1443#discussion_r1033123867

Reply:
Ah, you're right, I did misunderstand. "Individual" means "direct message" and "group" is referring to the chatrooms in the group. 👍 

IMO, it would be clearer to rename `CHATROOM_TYPES.INDIVIDUAL` to `CHATROOM_TYPES.DIRECT_MESSAGE`, but if you strongly prefer `CHATROOM_TYPES.INDIVIDUAL` we can leave it at that